tests/group/group_async_work_group_copy.cpp

```diff
@@ -89,38 +90,31 @@ class TEST_NAME : public util::test_base {
         "sycl::cl_ulong");
     for_type_and_vectors<check_type, sycl::cl_float>(log,
         "sycl::cl_float");
+#endif
 
 #ifdef INT8_MAX
-    if (!std::is_same<sycl::cl_char, std::int8_t>::value)
-      for_type_and_vectors<check_type, std::int8_t>(log, "std::int8_t");
+    for_type_and_vectors<check_type, std::int8_t>(log, "std::int8_t");
```

Comment:
This introduces duplication into the test. But you can move the if condition under `#if SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS`.

Reply:
Done, although this makes things a lot more verbose.